### ISSUE TITLE
Installed plugin no config generated

### ISSUE THREAD

**Issue by anesbit** [Valandur/Web-API#142]
Hello,

 I installed the plugin on my SF4 server and everything looks like its working it binds to the localhost and everything but when I goto sf4/configs/webapi all the files generated for the server a 1byte in size and empty.... any ideas on how to get this fixed?

**Comment by LonguCodes** [comment]
What engine are you running it on? Sponge?

**Comment by anesbit** [comment]
Yeah for sponge. To be fair I just decided to go on without it.. I have noticed a few updated versions since I may go back and try again.

**Comment by tsbreuer** [comment]
I ran into this same issue with spongeForge 7.3.0 and webapi 5.4.6-S7.2.0

**Comment by Valandur** [comment]
Hi, sorry for the late reply.

This is a duplicate of #146 , or that issue is a duplicate of this, however you would like to see it.

As mentioned in the other thread, I haven't worked on this library in a while so I'm not sure I can fix this issue.

I am however working on a complete rewrite with Sponge 8, Forge and Spigot support. If you can tell me your most wanted / used endpoints / features I can prioritize those and make sure to get a working version out soon-ish.
